Makefile

```diff
@@ -187,7 +187,12 @@ publish_mix: compile
 
 #==> Tests tasks
 
-test: test_erlang test_elixir
+test:
+	$(RM) -r .home
+	mkdir .home
+	HOME=$(CURDIR)/.home $(MAKE) test_erlang
```

Comment:
the original issue that motivated this is that running the tests creates `$HOME/.erlang.cookie` if it doesn't exist already (which is always the case on the debian autobuilders for example).